### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by kit1980**
Remove users that no longer have write access to the repo, resolving CODEOWNERS errors.

### CODE REVIEW

**Review by ZainRizvi — APPROVED** [review]

**Review by atalman — APPROVED** [review]

**Review by malfet — APPROVED** [review]
Please make sure to make similar changes to [docs/source/community/persons_of_interest.rst ](https://github.com/pytorch/pytorch/blob/main/docs/source/community/persons_of_interest.rst)

**Comment by malfet** [comment]
Cancelled merge, please make sure to move Zafar to emeritus maintainers in persons-of-interest

File: CODEOWNERS [hunk 1]
```diff
@@ -110,14 +110,8 @@ test/test_mps.py @kulinseth
 aten/src/ATen/mps/ @kulinseth
 aten/src/ATen/native/mps/ @kulinseth
 
-# Profiler
-torch/csrc/autograd/profiler* @robieta
-torch/autograd/profiler* @robieta
-torch/csrc/profiler/ @robieta
-torch/profiler/ @robieta
```
Comment (malfet) [inline CODEOWNERS:117]: Please replace @robieta with @aaronenyeshi

File: CODEOWNERS [hunk 2]
```diff
@@ -110,14 +110,8 @@ test/test_mps.py @kulinseth
...
-
 # AOTDispatch tests
 test/functorch/test_aotdispatch.py @ezyang @Chillee
 
 # Dataloader
-torch/utils/data/ @NivekT @ejguan
```
Comment (malfet) [inline CODEOWNERS:123]: Why do you remove Kevin here?

Comment (kit1980) [reply CODEOWNERS:123]: "Unknown owner on line 123: make sure @NivekT exists and has write access to the repository"

**Pull request closed without merging**